src/Templates/src/templates/maui-multihead/.template.config/template.json

```diff
@@ -0,0 +1,195 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "MAUI", "Android", "iOS", "macOS", "Mac Catalyst", "Windows", "Tizen", "Mobile" ],
```

Comment:
Should tizen be listed here and in the description? I don't see a separate Tizen project